cime_config/machines/config_compilers.xml

```diff
@@ -1795,12 +1807,9 @@ flags should be captured within MPAS CMake files.
     <append> -L$ENV{PNETCDF_PATH}/lib -lpnetcdf -L$ENV{HDF5_PATH}/lib -lhdf5_hl -lhdf5 -L$ENV{NETCDF_C_PATH}/lib -lnetcdf -L$ENV{NETCDF_FORTRAN_PATH}/lib -lnetcdff -L$ENV{ESSL_PATH}/lib64 -lessl -L$ENV{NETLIB_LAPACK_PATH}/lib -llapack</append>
   </SLIBS>
   <CXX_LINKER>FORTRAN</CXX_LINKER>
-  <!--
-  Summit (backup in case defaults are not picking up):
   <CXX_LIBS>
-    <base>/sw/summit/gcc/6.4.0/lib/gcc/powerpc64le-none-linux-gnu/6.4.0/crtbegin.o -L/sw/summit/gcc/6.4.0/lib64 -lstdc++ -latomic</base>
```

Comment:
I think @jgfouca added these a while ago. Should confirm with him if -latomic is still needed.

Reply:
@sarats , I didn't remember doing this so I went through old CIME history and it appears like you added `-latomic` back in this commit:

```
commit c88acfba6198e5c634871ee8527aad20c38f1d1b
Author: Sarat Sreepathi <sarat@ornl.gov>
Date:   Wed Jun 12 00:35:45 2019 -0400

    Summit: Add CXX libs for PGI build
    
    Need to add several CXX libs in flags to enable building with PGI.
```

Reply:
Thanks for refreshing my memory.